components/FilterSection/FilterSection.js

```diff
@@ -110,7 +110,7 @@ const StyledButton = styled.button`
   font-weight: bold;
 `;
 
-const StyledRuler = styled.hr`
+export const StyledRuler = styled.hr`
```

Comment:
hr should not be used for UI/UX design, rather for separating paragraph sections.
You could use the border bottom property. But if that is okay with your coach then don't mind my comment of course.

Apart from that `align-self` has no effect here. 
```suggestion
  display: flex-end;
```

Reply:
Thank you Laura 🥳